package.json

```diff
@@ -5,7 +5,7 @@
   "author": "Fuel Labs <contact@fuel.sh> (https://fuel.network/)",
   "private": true,
   "engines": {
-    "node": "^18.14.1",
+    "node": "^18.18.2",
```

Comment:
This needs to be updated in all `package.json` files to [ensure compatibility](https://github.com/FuelLabs/fuels-ts/pull/1338) for all packages.

Reply:
I created the #1399 PR to have this as part of the CI.

Reply:
As far as the comment goes, I resolved it in [9c08646](https://github.com/FuelLabs/fuels-ts/pull/1374/commits/9c086f63f24350f4d2d6dbf456a650ec4b86ec5f).

Reply:
Thank you.